sql/2022/media/video_adoption.sql

```diff
@@ -11,6 +12,10 @@ FROM (
     _TABLE_SUFFIX AS client,
     CAST(JSON_VALUE(JSON_VALUE(payload, '$._media'), '$.num_video_nodes') AS INT64) AS num_video_nodes
   FROM
-    `httparchive.pages.2022_06_01_*`)
+    `httparchive.pages.*`)
```

Comment:
This depends on the `media` custom metric, which didn't exist prior to the 2020 edition. So querying beyond that would waste a lot of bytes. 

```suggestion
    `httparchive.pages.*`
  WHERE
    _TABLE_SUFFIX >= '2020-08-01')
```